pkgs/crystal/package-bin.nix

```diff
@@ -13,7 +13,13 @@ lib.fix (compiler:
       passthru.buildCrystalPackage =
         callPackage ./build-crystal-package.nix {crystal = compiler;};
 
-      installPhase = ''
+      installPhase = if stdenv.isDarwin then
+      ''
+        mkdir -p $out/bin/
+        cp embedded/bin/crystal $out/bin/
+      ''      
+      else
+      ''
         mkdir -p $out
         cp -r bin lib share $out
```

Comment:
Also, why do we need lib and share for linux? So far in other places we only need _the binary_.